docs/recipes/install/common/warewulf4_setup_centos.tex

```diff
@@ -0,0 +1,29 @@
+% begin_ohpc_run
+% ohpc_comment_header Complete basic Warewulf setup for master node \ref{sec:setup_ww}
+\begin{lstlisting}[language=bash,literate={-}{-}1,keywords={},upquote=true,keepspaces]
+# Enable internal interface for provisioning
+[sms](*\#*) ip link set dev ${sms_eth_internal} up
+[sms](*\#*) ip address add ${sms_ip}/${internal_netmask} broadcast + dev ${sms_eth_internal}
+
+# Compute the network address for the internal network
+[sms](*\#*) internal_network=$(python3 -c "import ipaddress; print(str(ipaddress.IPv4Interface('$sms_ip/$internal_netmask').network.network_address))")
+
+# Edit the warewulf.conf file to use appropriate interface and settings
+[sms](*\#*) perl -pi -e "s/ipaddr:.*/ipaddr: ${sms_ip}/" /etc/warewulf/warewulf.conf
+[sms](*\#*) perl -pi -e "s/netmask:.*/netmask: ${internal_netmask}/" /etc/warewulf/warewulf.conf
+[sms](*\#*) perl -pi -e "s/network:.*/network: ${internal_network}/" /etc/warewulf/warewulf.conf
+[sms](*\#*) perl -pi -e 's/^(dhcp:.*)/$1\n  template: static/' /etc/warewulf/warewulf.conf
```

Comment:
This does not work as expected. There is now a line `template: static` and `template: default` and the second is used.

Reply:
Fixed code after testing against the right version of OpenHPC.